diff --git a/node_modules/antd/es/tooltip/index.js b/node_modules/antd/es/tooltip/index.js
index 5a2ae24..ae79c99 100644
--- a/node_modules/antd/es/tooltip/index.js
+++ b/node_modules/antd/es/tooltip/index.js
@@ -181,7 +181,7 @@ const InternalTooltip = /*#__PURE__*/React.forwardRef((props, ref) => {
     }),
     motion: {
       motionName: getTransitionName(rootPrefixCls, 'zoom-big-fast', props.transitionName),
-      motionDeadline: 1000
+      motionDeadline: 10
     },
     destroyTooltipOnHide: !!destroyTooltipOnHide
   }), tempOpen ? cloneElement(child, {
